src/api/data_types/chunking/compression.rs

```diff
@@ -0,0 +1,48 @@
+use std::fmt;
+
+use serde::{Deserialize, Deserializer};
+
+#[derive(Debug, Clone, Copy, Ord, PartialOrd, Eq, PartialEq, Default)]
+pub enum ChunkCompression {
```

Comment:
driveby question: how much effort would it be to add `zstd` compression to the whole workflow?

Reply:
I am not sure – have not worked too extensively with compression in the CLI yet.

What would be the benefit of adding `zstd`? Do we have any docs written about it?

Reply:
no, we don’t have any docs on that.
and `zstd` is pretty much the best choice if you want to trade compression ratio for resource usage.

Reply:
Ok, we should probably investigate how much effort this would be, will open an issue